src/helpers/helpers.ts

```diff
@@ -177,6 +177,19 @@ export const isValidURL = (url: string): boolean => {
   }
 };
 
+/**
+ * Checks if the current browser is Safari.
+ * @returns {boolean} Whether the browser is Safari.
+ */
+export const isSafari = () => {
+  const ua = navigator.userAgent.toLowerCase();
+  if (ua.indexOf('safari') !== -1) {
+    return ua.indexOf('chrome') === -1;
+  } else {
+    return false;
+  }
+};
```

Comment:
We can use the same as that comes from Starter.

```suggestion
export const isSafari = (): boolean => {
  // eslint-disable-next-line
  return /^((?!chrome|android).)*safari/i.test(navigator.userAgent) && !(window as any).MSStream;
};
```

Reply:
Done!